src/sentry/models/groupsubscription.py

```diff
@@ -60,14 +71,20 @@ def subscribe_actor(
         actor: Union[Team, User, RpcUser],
         reason: int = GroupSubscriptionReason.unknown,
     ) -> Optional[bool]:
+        from sentry import features
         from sentry.models import Team, User
 
         if isinstance(actor, RpcUser) or isinstance(actor, User):
             return self.subscribe(group, actor, reason)
         if isinstance(actor, Team):
-            # subscribe the members of the team
-            team_users_ids = list(actor.member_set.values_list("user_id", flat=True))
-            return self.bulk_subscribe(group, team_users_ids, reason)
+            if features.has(
+                "organizations:team-workflow-notifications", group.organization, actor=actor
```

Comment:
Do we need to check the flag for the actor? I think the actor kwarg might still expect a User instead of a team? 

If it's not explicitly necessary, we can skip it and it will [default](https://github.com/getsentry/sentry/blob/master/src/sentry/features/manager.py#L224) to `None`

Reply:
agree , i dont think we need it here